### PR TITLE
W-13888431: Add classifier and packaging as optional fields for depen…

### PR DESCRIPTION
**Pull request by llibarona**
…dencies

### CODE REVIEW

**Review by machaval — DISMISSED** [review]

File: exchange_api_packager/src/main/java/org/mule/maven/exchange/ValidateApiMojo.java [hunk 1]
```diff
@@ -49,8 +45,21 @@ public class ValidateApiMojo extends AbstractMojo {
...
+    private boolean skipValidateApi;
+
+
     @Override
     public void execute() throws MojoExecutionException, MojoFailureException {
+        if(skipValidateApi){
+            getLog().info("Ignoring validation..");
+            return;
+        } else {
+            getLog().info(String.format("To disable api validation parameterize '-D%s=true'", ApiProjectConstants.MAVEN_SKIP_VALIDATE_API));
```
Comment (machaval) [inline exchange_api_packager/src/main/java/org/mule/maven/exchange/ValidateApiMojo.java:61]: I wouldn't log this at info maybe at debug

**Review by machaval — DISMISSED** [review]

File: exchange-model/pom.xml [hunk 2]
```diff
@@ -5,7 +5,7 @@
     <parent>
         <artifactId>exchange_maven_client</artifactId>
         <groupId>org.mule.maven.exchange</groupId>
-        <version>2.3.0</version>
+        <version>2.3.1</version>
```
Comment (lbauret) [inline exchange-model/pom.xml:8]: Wouldn't make more sense to have a minor upgrade based on these changes?

**Review by lbauret — DISMISSED** [review]
LGTM!

**Review by marinarv — DISMISSED** [review]
LGTM!
If this PR also includes releasing a new version, there are some missing files. Check this [PR](https://github.com/mulesoft/api_project_maven_client/pull/64/files) as an example

**Comment by llibarona** [comment]
thx for the comment @marinarv ! I added the missing version changes

**Review by lbauret — APPROVED** [review]
LGTM!

**Review by marinarv — APPROVED** [review]
LGTM!

**Pull request merged**